### ISSUE TITLE
Telemetry table & map displayed incorrectly in Firefox

### ISSUE THREAD

**Issue by argilo** [projecthorus/radiosonde_auto_rx#915]
Upon loading Auto-RX in Firefox, the telemetry table is displayed with excessively wide columns, and the map is not centered:

![Screenshot from 2024-10-02 20-59-42](https://github.com/user-attachments/assets/9d096f81-289b-4881-aa66-8c2ebfb84d77)

Clicking a column heading restores the columns to their normal sizes, and resizing the browser window restores the map centering.

After some investigation, I traced the root cause to this line of code, which swaps Tabulator's dark theme for a light one:

https://github.com/projecthorus/radiosonde_auto_rx/blob/757925effaadf876152c1d42022e03b129c44594/auto_rx/autorx/templates/index.html#L70

If this line is commented out, then the problems go away (but the telemetry table always displays in dark mode).

If dark mode is selected in the settings pane, then the problems also go away and the page displays correctly upon initial load:

![Screenshot from 2024-10-02 21-00-00](https://github.com/user-attachments/assets/154d7327-b1ae-4b03-a7dc-5323592a769f)

I'm not yet sure what an appropriate fix would be.

**Comment by argilo** [comment]
I tried moving the stylesheet swap earlier (to the top of the main `<script>` block and out of `$( document ).ready`). After this change, Firefox sometimes (but not always) renders the table and map correctly. I expect the behaviour depends on how long the updated stylesheet takes to load.

I also tried updating Tabulator, and it appears that version 5.0 and later do not have this problem. However, there are a number of API changes in 5.0 that would need to be accounted for, and I haven't yet got the tables working.
